### ISSUE TITLE
Predicted output contains extra characters/missing character from tokens.

### ISSUE THREAD

**Issue by foramjivani** [wenwenyu/PICK-pytorch#103]
Thank you for the awesome solution. however, I am facing an error while predicting output as below : 
I have trained model for **name** and **address** key and I am getting validation accuracy for this as below : 

+--------------------+----------+----------+----------+----------+
| name               |      mEP |      mER |      mEF |      mEA |
+====================+==========+==========+==========+==========+
| address            | 0.920846 | 0.943102 | 0.931841 | 0.943102 |
+--------------------+----------+----------+----------+----------+
| name               | 0.965278 | 0.923588 | 0.943973 | 0.923588 |
+--------------------+----------+----------+----------+----------+

but when I predict I am getting following output
Rajesh Gangaram -- name

**Yadav** 374,shri ram chawk, omr road, chennai, tamil nadu, 600097 -- address

however **Yadav** is part of name. 

Can you please guide me how to reduce there token misclassifications ?

**Comment by Neelesh1121** [comment]
@foramjivani @wenwenyu same thing is happening with me also, please let me know if you have resolved this issue.
